### PR TITLE
type annotations added for aerospike_benchmark.py

### PR DESCRIPTION
**Pull request by ronaksingh27**
The following methods had their type annotations added

`GetConfig(user_config : Dict[str, Any] )  -> Dict[str, Any]` 
`Prepare(benchmark_spec: benchmark_spec.BenchmarkSpec) `
`Run(benchmark_spec: benchmark_spec.BenchmarkSpec) -> List[sample.Sample]`
`Cleanup(benchmark_spec: benchmark_spec.BenchmarkSpec) -> None`

### CODE REVIEW

File: perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py [hunk 1]
```diff
@@ -184,8 +186,8 @@
 """
 
 
-def GetConfig(user_config):
-  config = configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
+def GetConfig(user_config : Dict[str, Any] )  -> Dict[str, Any]:
+  config  = configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
```
Comment (jellyfishcake) [inline perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py:190]: There is an extra space here. Did you run the formatter? Can you run the formatter according to the beginner walkthrough?

- Check the changed file for formatting without changing the file:

  pyink --pyink-indentation 2 --pyink-use-majority-quotes --unstable --line-length=80 --check --diff {file_path}

- Format the file:

  pyink --pyink-indentation 2 --pyink-use-majority-quotes --unstable --line-length=80 {file_path}

**Review by jellyfishcake — CHANGES_REQUESTED** [review]
Looks great! Would appreciate if you can fix the extra whitespace issue!

**Review by jellyfishcake — APPROVED** [review]

File: perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py (hunk 1 above)
Comment (ronaksingh27) [reply perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py]: The space issue is resolved , should I raise PR for that ?

Yeah i already ran pytype 
Screenshot 2024-10-08 at 4 25 58 PM
No errors

**Comment by ronaksingh27** [comment]
Is there anything else I should take care of?

**Review by jellyfishcake — APPROVED** [review]

**Comment by jellyfishcake** [comment]
> Is there anything else I should take care of?
No this is all, we can pull this change now. Thank you!

**Review by bvliu — APPROVED** [review]

**Comment by ronaksingh27** [comment]
@bvliu @jellyfishcake Could you let me know why the `feedback/copybara — Google internal checks FAILED for runs with create time 2024-10-08T21:40:`  check was not successfull ?

**Comment by jellyfishcake** [comment]
> @bvliu @jellyfishcake Could you please let me know why the `feedback/copybara — Google internal checks FAILED for runs with create time 2024-10-08T21:40:` check was not successfull ?

Yes, this is an issue on our side. We will fix it from our side.

**Pull request merged**